.github/dependabot.yml

```
@@ -13,3 +13,9 @@ updates:
     reviewers:
       - stebenz
       - eliobischof
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

I'd configure the item similar to the one above.

```suggestion
  # Maintain dependencies for GitHub Actions
  - package-ecosystem: "github-actions"
    directory: "/"
    commit-message:
      prefix: "chore"
    schedule:
      interval: "weekly"
    groups:
      actions:
        patterns:
        - "*"
    reviewers:
      - stebenz
      - eliobischof
```

AFAIR, the groups section makes dependabot collect all updates in one PR, which IMO is more convenient.

---

Excellent suggestion, I've recreated the commit using your suggestion